backend/homework/7/src/main/java/org/example/vehicleservice/utils/MostExpAndCheap.java

```diff
@@ -0,0 +1,21 @@
+package org.example.vehicleservice.utils;
+
+import org.example.vehicleservice.models.Vehicle;
+
+import java.util.List;
+
+public class MostExpAndCheap {
+
+    private MostExpAndCheap(){}
+    public static Vehicle getMostExpensiveVehicle(List<Vehicle> vehicles){
+        return vehicles.stream()
```

Comment:
it is possible that the object will be null, so it's important to provide a proper message or handling mechanism for that scenario.